src/components/learner-credit-management/RequestStatusTableCell.jsx

```diff
@@ -49,6 +50,12 @@ const RequestStatusTableCell = ({ enterpriseId, row }) => {
     );
   }
 
+  if (lastActionErrorReason === 'failed_redemption') {
```

Comment:
I think we should use a constant value instead of hard coding. We can use those constants that we defined for BNR.